ui/jest.config.e2e.js

```diff
@@ -3,6 +3,11 @@ const baseJest = require('./jest.config.base.js');
 module.exports = {
   ...baseJest,
   preset: 'jest-playwright-preset',
-  testMatch: ['<rootDir>/e2e/test/*.e2e.js'],
+  testMatch: ['<rootDir>/e2e/**/*.feature', '<rootDir>/e2e/**/*.test.ts'],
```

Comment:
.ts for test step - or type script? I also dont see any files in here which have that suffix.

Reply:
Good spot, I think left over from my experimentation to get the tests to transform with gherkin. Removed.